src/Rootfs.jl

```diff
@@ -1100,3 +1102,88 @@ function installed_shards()
 
     return all_compiler_shards()[idx]
 end
+
+"""
+    manage_shards(; sort_by=:name, rev=false)
+
+Open a prompt allowing a user to selectively remove downloaded compiler shards.
+By default, the shards are sorted by name, alternatively you can sort them by
+file size on disk by specifying `sort_by=:size`. With `rev=true` you can
+reverse the sort order.
+"""
+function manage_shards(; sort_by=:name, rev=false)
+    # Get all installed shards
+    shards = installed_shards()
+
+    # Obtain directory sizes for the artifacts
+    totalsizes = totalsize.(Artifacts.artifact_path.(shard_source_artifact_hash.(shards)))
+
+    # Sort shards and totalsizes
+    if sort_by === :name
+        function isless_shards(a::CompilerShard, b::CompilerShard)
+            return isless((a.name, a.version, string(a.target), a.host, a.archive_type),
+                          (b.name, b.version, string(b.target), b.host, b.archive_type))
```

Comment:
I'm getting
```
julia> BinaryBuilderBase.manage_shards()
ERROR: MethodError: no method matching isless(::Platform, ::Platform)

Closest candidates are:
  isless(::Any, ::Missing)
   @ Base missing.jl:88
  isless(::Missing, ::Any)
   @ Base missing.jl:87

Stacktrace:
  [1] isless(t1::Tuple{Platform, Symbol}, t2::Tuple{Platform, Symbol})
    @ Base ./tuple.jl:541
  [2] isless
    @ ./tuple.jl:541 [inlined]
  [3] isless(t1::Tuple{VersionNumber, String, Platform, Symbol}, t2::Tuple{VersionNumber, String, Platform, Symbol})
    @ Base ./tuple.jl:541
  [4] isless(t1::Tuple{String, VersionNumber, String, Platform, Symbol}, t2::Tuple{String, VersionNumber, String, Platform, Symbol})
    @ Base ./tuple.jl:541
  [5] (::BinaryBuilderBase.var"#isless_shards#317")(a::CompilerShard, b::CompilerShard)
    @ BinaryBuilderBase ~/.julia/dev/BinaryBuilderBase/src/Rootfs.jl:1124
```

Reply:
Im afk, but what happens if you change `a.host` to `string(a.host)`?